knative-operator/pkg/controller/dashboard/sources/source_deployment_discovery_controller.go

```diff
@@ -0,0 +1,111 @@
+package sources
+
+import (
+	"context"
+
+	"github.com/openshift-knative/serverless-operator/knative-operator/pkg/common"
+	"k8s.io/api/apps/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+)
+
+const (
+	SourceLabel     = "eventing.knative.dev/source"
```

Comment:
since defined here too: https://github.com/openshift-knative/serverless-operator/pull/467/files#diff-96b561fd4c465bd07c16d9f0f812bdb1R34

I wonder... should we get somet constants like that upstream? 

I did not find one, doing a quick search: https://github.com/search?l=Go&q=org%3Aknative+%22eventing.knative.dev%2Fsource%22&type=Code

Reply:
Initially they were causing some cycle dependencies compilation failure due to how packages are imported.
But I can improve it.